src/mmw/js/src/analyze/views.js

```diff
@@ -364,6 +368,41 @@ var PointSourceTableView = Marionette.CompositeView.extend({
 
     onAttach: function() {
         $('[data-toggle="table"]').bootstrapTable();
+    },
+
+    ui: {
+        'pointSourceId': '.point-source-id'
+    },
+
+    events: {
+        'mouseover @ui.pointSourceId': 'showPointSourceMarker',
```

Comment:
I found it a little difficult to actually keep my mouse hovered over the link causing the point to flash on/off.  Given the suggestion about click/pan, how would you feel about the hover event being applied to the whole row to add it to the map (without pan)?